src/sentry/migrations/0165_alertruleowner.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 1.11.29 on 2021-02-22 16:07
+
+from django.conf import settings
+from django.db import migrations
+import django.db.models.deletion
+import sentry.db.models.fields.foreignkey
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = True
```

Comment:
This doesn't need to be dangerous. The models we're adding FKs to are safe, and just adding nullable columns to `AlertRule` should also be fine.

Reply:
Okay, nice. Would it be unsafe to add a FK to something like `Group`?